### PR TITLE
Generated Columns GA

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
doc updates for Generated Columns GA

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by ran-huang — APPROVED** [review]

File: develop/dev-guide-create-table.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ CREATE TABLE `bookshop`.`users` (
 
 - `{column_name}`: 列名。
 - `{data_type}`: 列的[数据类型](/basic-features.md#数据类型函数和操作符)。
-- `{column_qualification}`: 列的限定条件，如**列级约束**或[生成列（实验功能）](/generated-columns.md)子句。
+- `{column_qualification}`: 列的限定条件，如**列级约束**或[生成列](/generated-columns.md)子句。
```
Comment (qiancai) [inline develop/dev-guide-create-table.md:65]:
```suggestion
- `{column_qualification}`：列的限定条件，如**列级约束**或[生成列](/generated-columns.md)子句。
```

File: develop/dev-guide-create-table.md [hunk 2]
```diff
@@ -62,7 +62,7 @@ CREATE TABLE `bookshop`.`users` (
 
 - `{column_name}`: 列名。
 - `{data_type}`: 列的[数据类型](/basic-features.md#数据类型函数和操作符)。
```
Comment (qiancai) [inline develop/dev-guide-create-table.md:64]:
```suggestion
- `{column_name}`：列名。
- `{data_type}`：列的[数据类型](/basic-features.md#数据类型函数和操作符)。
```

**Review by qiancai — APPROVED** [review]
other LGTM

**Review by bb7133 — APPROVED** [review]
LGTM

**Pull request merged**